Documentation/distributions.md

```diff
@@ -0,0 +1,50 @@
+# Installing rkt on popular Linux distributions
+
+## CoreOS
+
+rkt is an integral part of CoreOS, installed with the operating system.
+The [CoreOS releases page](https://coreos.com/releases/) lists the version of rkt available in each CoreOS release channel.
+
+## Fedora
+
+rkt is packaged in Fedora Rawhide:
+```
+sudo dnf install rkt
+```
+
+The packaging work is tracked on the [Fedora package db page](https://admin.fedoraproject.org/pkgdb/package/rpms/rkt/).
```

Comment:
Until the rkt package makes its way into the general Fedora releases, [download the latest rkt directly from the project](https://github.com/coreos/rkt/releases).

rkt's entry in the [Fedora package database](https://admin.fedoraproject.org/pkgdb/package/rpms/rkt/) tracks packaging work for this distribution.